### ISSUE TITLE
Remember position in folder

### ISSUE THREAD

**Issue by OLEG4120** [SimpleMobileTools/Simple-Gallery#368]
It would be nice. Especially after device rotation

**Comment by tibbi** [comment]
you mean at the thumbnails view? The position is restored properly for me. The column count is often different, so you obviously wont always see the same images.

**Comment by OLEG4120** [comment]
Yeah, thumbnails view. Android 7.1.1 (moto g5s), after rotation gallery grop position to the top of the "list"

**Comment by OLEG4120** [comment]
When it is small folder (< 200 file), gallary remember position. If it is large folder (> 300 file) gallry dont do it
